transcendence-app/package.json

```diff
@@ -19,7 +19,9 @@
     "test:watch": "jest --watch",
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
-    "test:e2e": "jest --config ./test/jest-e2e.json"
+    "pretest:e2e": "docker compose -f docker-compose.test.yml up -d",
+    "test:e2e": "jest --config ./test/jest-e2e.json; ret=$?; npm run post:test:e2e; exit $ret",
+    "post:test:e2e": "docker compose -f docker-compose.test.yml down -v"
```

Comment:
Workaround to execute the ["post" script](https://docs.npmjs.com/cli/v6/using-npm/scripts#pre--post-scripts) even if the tests fail.